app.js

```diff
@@ -1,6 +1,4 @@
 require("dotenv").config();
-require("./db/connect");
```

Comment:
since `deta` is checking `index.js` file to run the project, we might have to load the DB here :thinking: